modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/mdf-request-completed-template/en-US.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html charset=UTF-8" />
+</head>
+
+<body>
+	<div class="header-logo" style="padding: 40px; text-align: center; margin: auto;"><img alt="Liferay Logo" class="logo"
```

Comment:
sort div css styles

Reply:
I thought for a moment that the gw formatSource indented the HTML lines of the templates. Fixed!

Reply:
Fixed

Reply:
all css classes should be sorted, not just line 10